doc-content/jbpm-docs/src/main/asciidoc/Kafka/kieserver-kafka-emit-proc.adoc

```diff
@@ -1,7 +1,7 @@
 [id='kieserver-kafka-emit-proc_{context}']
 = Configuring a service and the {KIE_SERVER} to emit Kafka messages when a transaction is committed
 
-You can configure the {KIE_SERVER} to emit Kafka messages about every event when a process, case, or task is completed. The {KIE_SERVER} sends the messages when it commits transactions. 
+You can configure the {KIE_SERVER} to emit Kafka messages about every event when a process, case, or task is completed and when the status of a task is changed. The {KIE_SERVER} sends the messages when it commits transactions.
```

Comment:
I would write, "every time a task, process or variable is created, updated or deleted, a corresponding view is published by the emitter when the transaction in which the modification occurs is completed"